CmrUI/Screens/NewsFeedTab/NewsFeedVM.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  NewsFeedVM.swift
+//  CmrUI
+//
+//  Created by Vasyl on 7/22/18.
+//  Copyright © 2018 Vasyl.Savka. All rights reserved.
+//
+
+import UIKit
+
+private struct Constants {
+    
+    let kImgCellHeight: CGFloat = 0.866
+    let kImgName = "pict_"
+    let backgroundColor = UIColor(red: 235.0/255.0, green: 235.0/255.0, blue: 235.0/255.0, alpha: 1.0)
+}
+
+protocol NewsFeedVMProtocol {
+    func updateDataSource()
+}
+
+class NewsFeedVM: NSObject {
+    
+        var dataSource = [FeedItem]()
```

Comment:
@vmsavka This is better, but it's not a flexible model accessor - you're assuming an array. The better way would be something like feedItemForRowAtItemIndex or something like this. (Internally, it goes to the array)

Reply:
fixed